template/title.phtml

```diff
@@ -0,0 +1,7 @@
+<?php

```

Comment:
This is not a template file so it can be called something like functions.php  - where you will have all functions that can be used on every page. Similar to the variables.php.

Reply:
all done